subprojects/smoke-test/build.gradle.kts

```diff
@@ -122,11 +119,35 @@ tasks {
         delete(remoteProjects.map { it.outputDirectory })
     }
 
-    withType<SmokeTest>().configureEach {
+    val santaSmokeTests = withType<SmokeTest>().matching { !it.name.startsWith("gradleBuild") }
```

Comment:
I would prefer doing the configuration when we register the smoke test task instead of relying on String matching here. WDYT?